docs/operating/troubleshooting.md

```diff
@@ -5,4 +5,22 @@ menu: operating
 slug: /troubleshooting.md
 ---
 
-Work in progress (:
+# Overlaps
+
+Overlaps should never happen in healthy and well configured Thanos system. That's why there is no automatic repair.
```

Comment:
Can we give example log line you see when you have overlaps? We should document each field in this log line as it can be quite tricky! (: WDYT?

Reply:
That would be helpful!
But i do not see any overlaps when use compator :( 
I would use logs in related issues.

Reply:
I only find those logs, could you provide more if possible?

Reply:
Just one is enough. But would be nice to expand it and explain. 

```msg="critical error detected; halting" err="compaction failed: compaction: pre compaction overlap check: overlaps found while gathering blocks. [mint: 1555128000000, maxt: 1555135200000, range: 2h0m0s, blocks: 2]: <ulid: 01D94ZRM050JQK6NDYNVBNR6WQ, mint: 1555128000000, maxt: 1555135200000, range: 2h0m0s>, <ulid: 01D8AQXTF2X914S419TYTD4P5B, mint: 1555128000000, maxt: 1555135200000, range: 2h0m0s>```

This example shows that you have really the same blocks for 2h overlaps. It worth checking are those blocks the same (check the number of series and samples, checksum the index and chunks).

Another example would be:
```
sg="critical error detected; halting" err="compaction failed: compaction: pre compaction overlap check: overlaps found while gathering blocks. [mint: 1555128000000, maxt: 1555135200000, range: 2h0m0s, blocks: 2]: <ulid: 01D94ZRM050JQK6NDYNVBNR6WQ, mint: 1555128000000, maxt: 1555135200000, range: 2h0m0s>, <ulid: 01D8AQXTF2X914S419TYTD4P5B, mint: 1555128000000, maxt: 1555142400000, range: 4h0m0s>
```

In such example we can explain that there are two blocks overlapping in 2h time range. One 4h second 2h block (: It's more tricky. This suggests someone uploaded compacted blocks rather. Again looking on meta.json would help.